server.py

```diff
@@ -0,0 +1,5 @@
+from waitress import serve
+from interface.wsgi import application
+
+if __name__ == '__main__':
```

Comment:
Hmmm, could we convert this to a django management command, and do `./manage.py waitress`?